how-to/onboarding-guide.md

```diff
@@ -1,81 +1,196 @@
+resources available to help you in that effort!
 
-The success of our peer review process is dependent upon a
-well-balanced editorial board. Our board needs to have combined expertise in:
+Below we discuss processes for finding and onboarding new volunteers to our
+peer review process.
 
-* A suite of specific science domains that fall within the scope of our peer review process
-* Technical expertise in Python packaging.
-* Awareness of the importance of documentation and package usability.
-* Awareness of the importance of CI / test suites to ensure robust software development.
+## About the Editorial Board
 
-## Recruiting new editors
+The success of our peer review process is dependent upon a well-balanced
+editorial board. Our board needs to have combined expertise in:
+
+* A suite of specific science domains that fall within the scope of our peer
+  review process
+* Technical expertise in Python packaging
+* Awareness of the importance of documentation and package usability
+* Awareness of the importance of CI / test suites to ensure robust software
+  development
+
+We also strive to ensure our editorial team is diverse and comprised of people
+from different backgrounds, cultures, genders, and domains.
+
+### Types of Editors
+
+#### New Editors Start as "Guests"
+
+A new editor will be considered a guest editor for the first 3 months of their
+tenure and/or until they have completed their first review. Once they have
+completed a review, they can be considered a full editor as deemed appropriate
+by the software review lead and the current editorial board.
+
+#### Adhoc Guest Editors
+
+Adhoc editors are editors with specific skill sets that are brought in to lead
+a single review. Examples of when there might be a need for an ad-hoc editor
+include:
+
+* If there is a conflict of interest between a package submitter and the
+  editorial team (e.g., a close colleague of everyone on the team)
+* If the editorial board is at capacity handling the current review load
+* If a very specialized skill set is needed (in a one-off type of situation)
+
+In this case, you may consider using our internal reviewer sign-up list to see
+if someone who signed up to be a reviewer might want to serve as an editor.
+
+### Experience Required to Be an Editor
+
+We prefer that editors have some experience with reviewing software. This
+experience could come from a previous review they worked on with pyOpenSci,
+rOpenSci, or JOSS.
+
+### Editorial Mentorship
+
+If a potential volunteer does not have prior software editorial experience, we
+offer a **mentorship process**. Editor mentorship is where someone with
+existing editorial experience mentors the new editor through their first
+review(s).
+
+### Recruiting New Editors
+
+Recruiting new editors and maintaining a sufficient and well-balanced
+editorial board is the responsibility of the
+[Software Review Lead](https://www.pyopensci.org/handbook/governance/structure.html#software-review-lead)
```

Comment:
flagging that it was earlier referred to as peer review lead - I'm assuming these are the same position?

Reply:
it is. i'll make it consistent locally!